### ISSUE TITLE
One SCSS File  Output Multiple CSS Files (Both Minified and Not Minified)

### ISSUE THREAD

**Issue by MarkSky** [NMFR/optimize-css-assets-webpack-plugin#32]
I write style in SCSS, and I want to use webpack to build both minified and not minified css files.
My webpack.config.js:
```
const path = require('path');
const webpack = require('webpack');
const ExtractTextPlugin = require('extract-text-webpack-plugin');
const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
const CssNano = require('cssnano');

const extractSASS = new ExtractTextPlugin({ filename: 'assets/styles/[name].css' });

module.exports = {
    entry: './Scripts/main.js',
    output: { path: path.resolve(__dirname, 'wwwroot/'), filename: 'scripts/bundle.js' },
    module: {
        rules: [
            {
                test: /\.scss$/,
                use: extractSASS.extract({
                    use: [
                        {
                            loader: 'css-loader',
                        },
                        {
                            loader: 'sass-loader',
                        },
                    ],
                    fallback: 'style-loader',
                }),
            },
            {
                test: /\.js$/,
                use: [{ loader: 'babel-loader', options: { presets: ['es2015'] } }],
            },
        ],
    },
    plugins: [
        new webpack.optimize.UglifyJsPlugin(),
        new OptimizeCssAssetsPlugin({
            assetNameRegExp: /\.optimize\.css$/g,
            cssProcessor: CssNano,
            cssProcessorOptions: {},
            canPrint: true,
        }),
        extractSASS,
    ],
};
```

Because it only build 1 css file, I would like to know how should I change to ouput 2 css files?
Thanks.

**Comment by NMFR** [comment]
See https://github.com/NMFR/optimize-css-assets-webpack-plugin/issues/29

TLDR: Not supported by the plugin out of the box but possible to achive with multiple parallel webpack configs.

I am starting to believe that this should indeed be a feature of the plugin.
I will start looking how to implement it.

**Comment by NMFR** [comment]
Also looking at your config you can also output 2 css files and use the `assetNameRegExp` option to make the plugin only modify one of them.

**Comment by MarkSky** [comment]
Excuse me, in my config, I only set to output one css:
`const extractSASS = new ExtractTextPlugin({ filename: 'assets/styles/[name].css' });`
But you said I can also output 2 css files.
I would like to know where to set output the other one?

**Comment by NMFR** [comment]
I don't recall what was the last suggestion but the first one was to generate 2 CSS files by using multiple parallel webpack configs:

```
module.exports = [
  {
    // webpack config without OptimizeCssAssetsPlugin to produce the original CSS file
  },
  {
    // webpack config with OptimizeCssAssetsPlugin to produce the optimized CSS file
  }
];
```

**Comment by MarkSky** [comment]
@NMFR Thank you. It's good idea to output 2 css files.
